gammapy/detect/cwt.py

```diff
@@ -108,34 +139,48 @@ class CWT(object):
         - Fill ``self.model += res * (res > 0)``
         - What is this??
 
-
     - max_scale_image : 2D image, value = index of scale where emission is most significant
 
-    Parameters
+    Examples
+    --------
+    TODO
+
+    References
     ----------
-    min_scale : float
-        first scale used
-    nscales : int
-        number of scales considered
-    scale_step : float
-        base scaling factor
+    R. Terrier, L. Demanet, I.A. Grenier, and J.P. Antoine. Wavelet
+    analysis of EGRET data. Universite   Paris VII & Sap CEA Saclay.
+    Institut de physique theorique et mathematique, Universite
+    catholique de Louvain, Louvain-la-Neuve
```

Comment:
The most important thing to add is the link to http://adsabs.harvard.edu/abs/2001ICRC....7.2923T so that others can quickly find what publication exactly you are referencing.
The description can be shortened to only contain the key information like this:
```
R. Terrier et al (2001) "Wavelet analysis of EGRET data"
See http://adsabs.harvard.edu/abs/2001ICRC....7.2923T
```